test/slack/lib.test.js

```
@@ -0,0 +1,117 @@
+                    username: 'MySlackBot',
+                    channelId,
+                    text: message
+                }
+            });
+        });
+
+        it('should throw an error when not using AuthHub and botToken is not available', async () => {
+            const channelId = 'testChannelId';
+            const message = 'testMessage';
+            context.config.usesAuthHub = undefined;
+            delete context.config.botToken;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)